prodsys/express/resources.py

```diff
@@ -38,7 +38,7 @@ class Resource(ABC):
     location: conlist(float, min_items=2, max_items=2) # type: ignore
     capacity: int = 1
     states: Optional[List[state.STATE_UNION]] = Field(default_factory=list)
-    controller: resource_data.ControllerEnum = resource_data.ControllerEnum.PipelineController
+    controller: resource_data.ControllerEnum = Union [resource_data.ControllerEnum.PipelineController, resource_data.ControllerEnum.BatchController]
```

Comment:
Initial value should not be a Union -> This is wrong. Just remain the line as before.